sdk/src/main/java/com/uber/sdk/rides/client/model/RideReceipt.java

```diff
@@ -0,0 +1,148 @@
+
+package com.uber.sdk.rides.client.model;
+
+import java.util.List;
+
+import javax.annotation.Nullable;
+
+/**
+ * A receipt for a completed request.
+ * See
+ * <a href="https://developer.uber.com/docs/v1-requests-receipt">Ride Request Receipt</a>
+ * for more information.
+ */
+public class RideReceipt {
+    private String request_id;
+    private List<Charge> charges;
+
+    @Nullable
+    private Charge surge_charge;
+
+    private List<Charge> charge_adjustments;
+    private String normal_fare;
+    private String subtotal;
+    private String total_charged;
+
+    @Nullable
+    private Float total_owed;
+
+    private String currency_code;
+    private String duration;
+    private String distance;
+    private String distance_label;
+
+    /**
+     * The unique ID of the ride.
+     */
+    public String getRequestId() {
+        return request_id;
+    }
+
+    /**
+     * The charges made against the rider.
```

Comment:
Yes, I know about that. However I noticed that most model classes already in the project do not follow such guidelines, that's why I used the same style as the majority, to keep it "consistent".

There are 5 classes in the model package that follow the guidelines: PaymentMethod, PaymentMethodsResponse, Place, RideRequestParameters, RideUpdateParameters.
The other 20 classes all use the style is used in this PR.

I've changed the method descriptions to start with a verb.